content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Customizing Istio Metrics
+description: This task shows you how to customize the Istio metrics.
+weight: 25
+keywords: [telemetry,metrics,customize]
+---
+
+This task shows you how to customize the metrics that Istio generates.
+
+Istio generates telemetry that various dashboards consume to help you visualize
+your mesh. For example, dashboards that support Istio include:
+
+* [Grafana](/docs/tasks/observability/metrics/using-istio-dashboard/)
+* [Kiali](/docs/tasks/observability/kiali/)
+* [Prometheus](/docs/tasks/observability/metrics/querying-metrics/)
+
+By default, Istio defines and generates a set of standard metrics (e.g.
+`requests_total`), but you can also customize them and create new metrics.
+
+## Custom statistics configuration
+
+Istio uses the Envoy proxy to generate metrics and provides its configuration in
+the `EnvoyFilter` at
+[`manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml).
+
+Configuring custom statistics involves two sections of the
+`EnvoyFilter`: `definitions` and `metrics`. The `definitions` section
+supports creating new metrics by name, the expected value expression, and the
+metric type (`counter`, `gauge`, and `histogram`). The `metrics` section
+provides values for the metric dimensions as expressions, and allows you to
+remove or override the existing metric dimensions. You can modify the standard
+metrics using `tags_to_remove` or by re-defining a dimension.
```

Comment:
nit: should this say: "standard metrics dimensions" instead of just "standard metrics"

Reply:
fixed